backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -15,6 +15,11 @@ include::snippets/technology-preview.adoc[]
 
 If your cloud provider has a native snapshot API or supports xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI) snapshots], the `Backup` CR backs up persistent volumes by creating snapshots. For more information, see the xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Overview of CSI volume snapshots] in the {product-title} documentation.
 
+[IMPORTANT]
+====
```

Comment:
OADP is not spelled out in its full term in this module. Should it be spelled out fully on its first occurrence?

Reply:
Spell out acronyms on the first occurrence in each module and assembly.